ga/latest/full/Dockerfile.ubi.adoptopenjdk11

```diff
@@ -21,12 +21,11 @@ RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
   && installUtility install --acceptLicense baseBundle \
   && if [ ! -z $REPOSITORIES_PROPERTIES ]; then rm /opt/ibm/wlp/etc/repositories.properties; fi \
   && rm -rf /output/workarea /output/logs \
-  && chmod -R g+rwx /opt/ibm/wlp/output/*
+  && find /opt/ibm/wlp ! -perm -g=rw -print0 | xargs -r -0 chmod g+rw
 
 COPY --chown=1001:0 server.xml /config/
 
 # Create a new SCC layer
 RUN if [ "$OPENJ9_SCC" = "true" ]; then populate_scc.sh; fi \
     && rm -rf /output/messaging /output/resources/security /logs/* $WLP_OUTPUT_DIR/.classCache \
-    && chown -R 1001:0 /opt/ibm/wlp/output \
```

Comment:
Generally, we do set this so that all files are of group `0` as that what the container runs as in kubernetes.

eg:
sh-4.4$ id
uid=1000680000(1000680000) gid=0(root) groups=0(root),1000680000

Reply:
If the output and files are created as root:root that should be ok as long as we have group permissions correctly.
In standalone docker scenario it still runs as:
uid=1001(default) gid=0(root) groups=0(root)

Reply:
I think everything running here will be running as user 1001.